sources/api/migration/migrations/v1.1.0/kubelet-settings-prefixes-migration/Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "kubelet-settings-prefixes-migration"
```

Similar to my comment above, the package name is too generic.
```suggestion
name = "kubelet-kube-reserved-and-eviction-hard"
```